app/utils/load-helpscout-beacon.js

```
@@ -0,0 +1,34 @@
+        beaconScript.type = 'text/javascript';
+        beaconScript.async = true;
+        beaconScript.src = 'https://beacon-v2.helpscout.net';
+        firstScript.parentNode.insertBefore(beaconScript, firstScript);
+        beaconScript.onload = resolve;
+      }
+
+      if (
+        ((window.Beacon = Beacon =
+          function (method, options, data) {
+            window.Beacon.readyQueue.push({ method, options, data });
+          }),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)